Migration.md

```diff
@@ -113,6 +113,9 @@ one root level model, actor or light.
    respectively. Users are encouraged to use the constants `kSdfStringSource` 
    and `kUrdfStringSource` instead of hard-coding the string literals.
 
+1. USD component now is living in https://github.com/gazebosim/gz-usd as an
+   independent package.
```

Comment:
Can we create a new section for this above, called `## libsdformat 12.5.0 to 12.6.0`?

Reply:
https://github.com/gazebosim/sdformat/pull/1094/commits/f3daf6e9c744d7d1fde87ccf13ca76695f79e008